index.html

```diff
@@ -18,10 +18,18 @@
             <div class="logo_nav">
                 <a href="#"><img class="image_nav" src="images/my_logo.svg" alt="logo"></a>
             </div>
-            <a href="#"><img class="menu_icon" src="images/menu.svg" alt="menu"></a>
+            <div class="nav_content">
+                <a href="#"><img class="menu_icon" src="images/menu.svg" alt="menu"></a>
+                <ul class="nav_menu">
+                    <li><a class="nav_li_item" href="#portfolio">Portfolio</a></li>
+                    <li><a class="nav_li_item" href="#about_me">About me</a></li>
+                    <li><a class="nav_li_item" href="#contact">Contact</a></li>
+                </ul>
+            </div>
         </nav>
         <section class="first_section">
-            <h1 class="title content">I’m Alvaro. Glad to see you!</h1>
+            <div class="content_header">
+            <h1 class="title content">I’m Alvaro. <br> Glad to see you!</h1>
```

Comment:
Please consider using the same styles for the paragraph as being used in the design like `letter spacing, font-size, etc` to make it look exactly like the design.

## Note:- 

The problem exists in the mobile version. Remember, your mobile version also need to be exactly similar to the design in this project so consider reviewing it also.

## Solution:-

Remember, to copy the styles of an element click on it in Figma and copy the styles from the right panel or side bar.